submissions/natashafir/js-dom/index.js

```diff
@@ -0,0 +1,108 @@
+const info = [
+    {
+        id: 0,
+        name: 'Ciudad de México',
+        img_url: './img/cdmx.jpeg'
+    },
+    {
+        id: 1,
+        name: 'Guadalajara',
+        img_url: './img/gdl.jpg'
+    },
+    {
+        id: 2,
+        name: 'Cancún',
+        img_url: './img/cancun.jpg'
+    },
+    {
+        id: 3,
+        name: 'Tulum',
+        img_url: './img/tulum.jpg'
+    },
+    {
+        id: 4,
+        name: 'Los Colorados',
+        img_url: './img/colorados.jpg'
+    },
+    {
+        id: 5,
+        name: 'Puebla',
+        img_url: './img/puebla.jpg'
+    },
+    {
+        id: 6,
+        name: 'La Playa del Amor',
+        img_url: './img/amor.jpg'
+    },
+    {
+        id: 7,
+        name: 'Cenote Suytun',
+        img_url: './img/suytun.jpg'
+    },
+    {
+        id: 8,
+        name: 'Tolantongo',
+        img_url: './img/tolantongo.jpg'
+    },
+    {
+        id: 9,
+        name: 'Oaxaca',
+        img_url: './img/oaxaca.jpg'
+    },
+    {
+        id: 10,
+        name: 'La Paz',
+        img_url: './img/la-paz.jpg'
+    },
+    {
+        id: 11,
+        name: 'Playa Mujeres',
+        img_url: './img/mujeres.jpg'
+    }
+
+];
+
+let places = document.getElementById('main');
+let navItems = document.getElementById('nav__wrapper');
+let img = document.createElement('img');
+let nav = document.getElementById("navigation");
+let burger = document.getElementById("hamburger");
+
+img.src = info[0].img_url;
+places.appendChild(img);
+
+info.forEach(function (element, index) {
```

Comment:
Create a function something like `drawMenuList` and call in function after load a page in block `window.onload` or `document.addEventListener("DOMContentLoaded", ......);`